index.html

```diff
@@ -7,5 +7,200 @@
 </head>
 <body>
 
+<!-------------------------------------------------- Content header --------------------------------------------------->
```

Comment:
Fijne structuur in je HTML en CSS file! Dit is heel goed om te doen, het geeft je overzicht en maakte het voor mij fijn leesbaar.